MomentumDiscordBot/Commands/Checks/RequireAdminBotChannelAttribute.cs

```diff
@@ -9,7 +9,7 @@ public class RequireAdminBotChannelAttribute : DescriptiveCheckBaseAttribute
     {
         public RequireAdminBotChannelAttribute() => FailureResponse = "Requires the bot admin channel";
 
-        public override Task<bool> ExecuteCheckAsync(CommandContext context, bool help)
+        public override Task<bool> ExecuteChecksAsync(InteractionContext context)
```

Comment:
Was the `help` bool doing any significant here?

Reply:
The CheckBaseAttribute abstract class required it but it wasn't used. I changed the base class here and adjusted the override.